test/e2e/query_test.go

```diff
@@ -761,9 +761,7 @@ func TestSidecarQueryEvaluation(t *testing.T) {
 			testutil.Ok(t, synthesizeSamples(ctx, prom2, tc.prom2Samples))
 
 			testQuery := func() string { return tc.query }
-			queryAndAssert(t, ctx, q.Endpoint("http"), testQuery, time.Now, promclient.QueryOptions{
-				Deduplicate: true,
```

Comment:
This wasn't actually testing deduplication since both Prometheus nodes were called differently